SoftwareGuide/Latex/Introduction/Installation.tex

```diff
@@ -438,6 +438,49 @@ \subsection{Advanced Module Configuration}
 displayed (Figure \ref{fig:ConfigITKDefault}); these messages are sorted in
 alphabetical order by module names.
 
+Those who prefer to build ITK using the command line are referred to the online
+cmake command-line tool
+documentation\footnote{\url{https://cmake.org/cmake/help/latest/manual/cmake.1.html}.
+Only some typical use cases are shown here for reference.
+
+\begin{itemize}
+\item \textbf{Example 1}: Build all default modules.
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+cmake [-DITK_BUILD_DEFAULT_MODULES:BOOL=ON]
+      /ITK-build
+\end{minted}
+\normalsize
+
+The default is \code{ITK\_BUILD\_DEFULT\_MODULES=ON}, so it is optional to
```

Comment:
As \code{ITK\_BUILD\_DEFULT\_MODULES} is \code{ON} by default, the above can also be accomplished by
\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
cmake ../ITK-build
\end{minted}

Reply:
The sentence is a bit confusing in its current form. We could also omit the long form with explicitly passing the `ITK_BUILD_DEFULT_MODULES`.